### ISSUE TITLE
LDAP is not sending authentication requests on login.

### ISSUE THREAD

**Issue by Azerothian** [ntop/ntopng#784]
LDAP is not sending authentication requests on login.

I am using docker to build and deploy the application please see the following for my setup.
https://github.com/RungeICT/ntopng-docker/blob/master/Dockerfile

We have a valid ntopng pro license  e.g. Generated by ntopng Pro [Small Business Edition] v.2.4.161004

Attempting to query against Samba 4 Active Directory Domain Controller with the following options enabled on the global entry (allows for loose tls connections)

```
        requirecert = false
        client ldap sasl wrapping = sign
        ldap server require strong auth = no
```

Nothing appears in the log file when attempting to login.

Has the stable branch been compiled correctly with LDAP Flag?
https://github.com/ntop/ntopng/blob/97e9f98c00eb408d4d2234b035823980e14d170b/src/Ntop.cpp#L686

**Comment by lucaderi** [comment]
We have just realised that Ubuntu 16 packages did not have LDAP support included. We have rebuilt the package with LDAP support. If this is you case please upgrade and report. If not your case (you apparently use 14.04) please send us ntopng --version

**Comment by simonemainardi** [comment]
also make sure to rebuild the docker image without caching with option `--no-cache`

**Comment by mattdunn4465** [comment]
I am using 14.04 and I don't see any packets attempting to hit the ldap host when you try to login to ntop.

**Comment by simonemainardi** [comment]
@fullmoon4465 LDAP is a pro-only feature. Please refer to this readme https://github.com/ntop/ntopng/blob/dev/doc/README.LDAP and make sure to properly configure the setting from the ntopng preferences

**Comment by simonemainardi** [comment]
fixed in : https://github.com/ntop/ntopng/commit/aa8789db7f282e5106a9dd53a9cdec791b782277

**Comment by BroadbandCommunicationsNorth** [comment]
We have the same issue with 2.4.161220 - Pro Small Business Edition, CentOS Linux release 7.3.1611 (Core). 
According tcpdump, Ntop does not send any traffic to Active Directory Ldap server, which is set up in ldap://10.10.10.1:389. Tried to include verbose mode for Ntopng, nothing appeared in the log.

**Comment by simonemainardi** [comment]
@BroadbandCommunicationsNorth fix is contained in version 2.5
